src/components/MapController/MapController.js

```diff
@@ -150,9 +150,23 @@ var MapController = React.createClass({
           onSetArea={this.handleSetArea}
           area={this.state.area}
         />
+      );    
+      var dataAttributes = this.state.dataset.split('_');
+      var run = dataAttributes[4];
```

Comment:
This is more of a discussion question than a comment, but is it valuable to have the variable shown on the footer? I guess that's something that the user will have selected (and will be shown above), but is there value to having it be explicitly shown on the map as well? (For future map printing, or for screenshots?)